sorts/src/test/java/za/co/oldtimerza/sorts/InsertionSortTests.java

```diff
@@ -0,0 +1,27 @@
+package za.co.oldtimerza.sorts;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class InsertionSortTests {
```

Comment:
Use the convention SomethingTest instead of SomethingTests .

classes ending in Test are automatically picked up by JUnit by default.